.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
     - "0.10"
-    - "0.12.3"
+    - "0.12"
```

Comment:
I believe that if the two node versions specified here have different specificity levels, travis can't figure out which version to use. This is what happened in the failed build in the previous commit.